index.bs

```diff
@@ -3552,73 +3552,49 @@ Methods</h4>
-		<code>cancelTime</code> is reached is quite complicated. The
-		behavior of {{AudioParam/cancelAndHoldAtTime()}} is therefore
-		specified in the following algorithm.
-
-		<div algorithm="cancel and hold">
-			Let \(t_c\) be the value of <code>cancelTime</code>. Then
-
-			1. Let \(E_1\) be the event (if any) at time \(t_1\) where
-				\(t_1\) is the largest number satisfying \(t_1 \le t_c\).
-
-			2. Let \(E_2\) be the event (if any) at time \(t_2\) where
-				\(t_2\) is the smallest number satisfying \(t_c \lt t_2\).
-
-			3. If \(E_2\) exists:
-				1. If \(E_2\) is a linear or exponential ramp,
-					1. Effectively rewrite \(E_2\) to be the same kind of
-						ramp ending at time \(t_c\) with an end value that
-						would be the value of the original ramp at time
-						\(t_c\).
-						<img alt="Graphical representation of calling cancelAndHoldAtTime when linearRampToValueAtTime has been called at this time." src="images/cancel-linear.svg" style="height:75%;width:75%">
-
-					2. Go to step 5.
-
-				2. Otherwise, go to step 4.
-
-			4. If \(E_1\) exists:
-				1. If \(E_1\) is a <code>setTarget</code> event,
-					1. Implicitly insert a <code>setValueAtTime</code>
-						event at time \(t_c\) with the value that the
-						<code>setTarget</code> would have at time
-						\(t_c\).
-						<img alt="Graphical representation of calling cancelAndHoldAtTime when setTargetAtTime has been called at this time" src="images/cancel-setTarget.svg" style="height:75%;width:75%">
-
-					2. Go to step 5.
-
-				2. If \(E_1\) is a <code>setValueCurve</code> with a start
-					time of \(t_3\) and a duration of \(d\)
-
-					1. If \(t_c \gt t_3 + d\), go to step 5.
-
-					2. Otherwise,
-						1. Effectively replace this event with a
-							<code>setValueCurve</code> event with a start time
-							of \(t_3\) and a new duration of \(t_c-t_3\).
-							However, this is not a true replacement; this
-							automation MUST take care to produce the same
-							output as the original, and not one computed using
-							a different duration. (That would cause sampling of
-							the value curve in a slightly different way,
-							producing different results.)
-							<img alt="Graphical representation of calling cancelAndHoldAtTime when setValueCurve has been called at this time" src="images/cancel-setValueCurve.svg" style="height:75%;width:75%">
-
-						2. Go to step 5.
-
-			5. Remove all events with time greater than \(t_c\).
-
-			If no events are added, then the automation value after
-			{{AudioParam/cancelAndHoldAtTime()}} is the constant value that
-			the original timeline would have had at time \(t_c\).
-		</div>
+		When <code>cancelAndHoldAtTime</code> is called with a time <var>t</var> as
+		a parameter, execute the following steps:
+
+		- Let <var>v</var> be the value of the {{AudioParam}} at time
+			<var>t</var>.
+		- Let <var>current</var> be the <a lt="current audioparam">current
+			AudioParam</a> at time <var>t</var>.
```

Comment:
I guess you meant "current AudioParam event"? I think it's better to say "current automation event" (the term is defined in the spec).

Also, you only use _current_ once after you define it, so maybe you can just use "current automation event" instead.